From 0ef24a829eeedfd9986ede14ac1cfadcdcc49af0 Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Fri, 23 Feb 2018 13:31:33 +0800
Subject: [PATCH 1/2] yaffs2: include rawnand.h instead of nand.h

The nand.h has been renamed to rawnand.h by commit d4092d76a4a4 ("mtd:
nand: Rename nand.h into rawnand.h"), adjust the yaffs2 code according
to this change.

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 fs/yaffs2/yaffs_mtdif.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/yaffs2/yaffs_mtdif.c b/fs/yaffs2/yaffs_mtdif.c
index fa4b99834570..03a3d9c24e4c 100644
--- a/fs/yaffs2/yaffs_mtdif.c
+++ b/fs/yaffs2/yaffs_mtdif.c
@@ -18,7 +18,7 @@
 #include "linux/mtd/mtd.h"
 #include "linux/types.h"
 #include "linux/time.h"
-#include "linux/mtd/nand.h"
+#include "linux/mtd/rawnand.h"
 #include "linux/kernel.h"
 #include "linux/version.h"
 #include "linux/types.h"
-- 
2.5.0

